build.gradle

```diff
@@ -126,6 +126,14 @@ allprojects {
         exclude group: 'ch.qos.logback', module: 'logback-classic'
         exclude group: 'ch.qos.logback', module: 'logback-core'
     }
+    normalization {
+        runtimeClasspath {
+            properties {
+                ignoreProperty 'version.date'
+            }
+            ignore '**/views.properties'
```

Comment:
so just to clarify - all `view.properties` files are ignored as well as any property named `version.date`. the `version.date` property is not in the `views.properties` file correct?

Reply:
Correct, the views.properties has a comment in it with a timestamp.  If this information needed to have changes detected, then we would have to find a new way to create / generate the views.properties file without that comment.  That comment seems to be standard functionality for java.util.Properties created files.